pkg/cache/config.go

```diff
@@ -0,0 +1,200 @@
+package cache
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	cbuilder "github.com/scribd/go-sdk/internal/pkg/configuration/builder"
+)
+
+type (
+	StoreType int
+
+	// Redis provides configuration for redis cache.
+	Redis struct {
+		// URL into Redis ClusterOptions that can be used to connect to Redis
+		URL string `mapstructure:"url"`
+
+		// Either a single address or a seed list of host:port addresses
+		// of cluster/sentinel nodes.
+		Addrs []string `mapstructure:"addrs"`
```

Comment:
[nit]: Isn't `Addrs` a bit confusing? I guess lazy people like me would miss the `e` in middle. I suggest using `Address` or `Addr`, wdyt?

Reply:
@laynax my guess is that @Neurostep is mapping the fields [already used by the UniversalClient](https://github.com/redis/go-redis/blob/v9.5.1/universal.go#L13-L15).

Reply:
@fotos Ah right, it's an array meaning `addresses`. I'll mark this as resolved 😅